ride-cleansing/src/main/java/org/apache/flink/training/exercises/ridecleansing/RideCleansingMapExercise.java

```diff
@@ -0,0 +1,77 @@
+package org.apache.flink.training.exercises.ridecleansing;
+
+import org.apache.flink.api.common.JobExecutionResult;
+import org.apache.flink.api.common.functions.MapFunction;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.configuration.ConfigurationUtils;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.streaming.api.functions.sink.PrintSinkFunction;
+import org.apache.flink.streaming.api.functions.sink.SinkFunction;
+import org.apache.flink.streaming.api.functions.source.SourceFunction;
+import org.apache.flink.training.exercises.common.datatypes.EnrichedRide;
+import org.apache.flink.training.exercises.common.datatypes.TaxiRide;
+import org.apache.flink.training.exercises.common.sources.TaxiRideGenerator;
+
+import java.util.Properties;
+
+/**
+ * The Ride Cleansing exercise from the Flink training.
+ *
+ * <p>The task of this exercise is to filter a data stream of taxi ride records to keep only rides
+ * that both start and end within New York City. The resulting stream should be printed.
+ */
+public class RideCleansingMapExercise {
```

Comment:
Sorry, I don't understand purpose of this class.
Is it something additional from your side which you decided to try?

Reply:
yes this is an additional one

Reply:
ok